apps/web/src/components/Radio.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Field, ErrorMessage } from 'formik';
+
+export interface FormFieldProps {
+  label?: string;
+  name: string;
+  options: Array<{
+    key: string;
+    value: string | boolean;
+  }>;
+}
+
+export const renderBooleanOrString = (value: string | boolean) => {
+  if ('boolean' === typeof value) {
+    return value ? 'Yes' : 'No';
+  }
+  return value;
+};
+
+function RadioButtons(props: FormFieldProps) {
+  const { label, name, options, ...rest } = props;
+  return (
+    <div>
+      <label>{label}</label>
+      <Field name={name}>
+        {formik => {
+          const { field } = formik;
+          return options.map(option => {
+            return (
+              <div
+                className='flex space-x-5 items-center cursor-pointer leading-none py-3'
+                key={option.key}
+              >
+                <input
```

Comment:
read my previous concern

Reply:
I am going to leave this for now and address this change in an upcoming PR, to focus more on the flow now